frontend/src/views/supp-allowances/SupplementaryFormView.vue

```diff
@@ -105,12 +163,36 @@ export default {
       },
     ]
     this.panel = this.allPageIDs
+    await this.loadData()
+    this.loading = false
   },
   methods: {
+    ...mapActions(useApplicationsStore, ['getSupplementaryApplications']),
     isEmpty,
     togglePanel() {
       this.panel = isEmpty(this.panel) ? this.allPageIDs : []
     },
+    async loadData() {
+      this.setUpDefaultNewRequestModel(await this.getSupplementaryApplications(this.applicationId))
+    },
+    setUpDefaultNewRequestModel(suppApplications) {
+      let indigenousProgrammingModel = {
+        indigenousFundingModel: [],
+        indigenousOtherDescription: null,
+        supplementaryApplicationId: undefined,
+        supplementaryType: 2,
+      }
+
+      let supportModel = { test: 'test', supplementaryType: 1 }
+      let transportModel = { test: '', supplementaryType: 3 }
+
+      this.transportModel = findAndUpdateModel(suppApplications, transportModel)
+      this.indigenousProgrammingModel = findAndUpdateModel(suppApplications, indigenousProgrammingModel)
+      this.supportModel = findAndUpdateModel(suppApplications, supportModel)
+    },
+    updateIndigenousModel(updatedModel) {
+      this.indigenousProgrammingModel = updatedModel //funding model is an array
+    },
```

Comment:
do we need a try catch here and show an alert using the alertMixin for the user? In case we aren't able to load up the supplementaryApplication? 

example https://github.com/bcgov/ecc-ofm/blob/a4dbf3cf516e17850adad10e49860262c683c031/frontend/src/views/ReportingView.vue#L235C9-L235C119